code/game/machinery/floodlight.dm

```diff
@@ -59,7 +61,7 @@
 			return
 		on = 1
 		user << "\blue You turn on the light"
-		set_light(brightness_on)
+		set_light(brightness_range, brightness_power, light_color)
```

Comment:
I settled on buffing the light power one notch, while keeping the range unchanged. Light color's been removed.